torch/distributed/_composable/fsdp/_fsdp_state.py

```diff
@@ -126,6 +131,13 @@ def _lazy_init(self) -> None:
         for state in self._state_ctx.all_states:
             if state._fsdp_param_group:
                 state._fsdp_param_group.lazy_init()
+                for param in state._fsdp_param_group.fsdp_params:
+                    logger.debug(
+                        "FSDP::lazy_init, fqn=%s, dtype=%s, shape=%s",
+                        param._param_fqn,  # noqa: W291
+                        param.sharded_param.dtype,  # noqa: W291
+                        param._orig_size,
+                    )
```

Comment:
@weifengpy Do we want to include this in this PR?
I am not convinced by why we need this info of FQN, dtype, and original size. These are all available to the user before applying FSDP.

```
for param_name, param in model.named_parameters():
    print(param_name, param.dtype, param.shape)
```

Reply:
we can remove this from the PR